db/seeds.rb

```diff
@@ -22,7 +15,17 @@
 Activity.create(name: "Hiking", description: "We loven to do ze hikes.", image_url: "hiker.jpg" )
 Activity.create(name: "Rafting", description: "Rafting Rulez!", image_url: "raft.jpg" )
 Activity.create(name: "Scuba Diving", description: "Scuba Steve: Damn you!.", image_url: "scuba.jpg" )
+Activity.create(name: "Climbing", description: "bloody hands.", image_url: "scuba.jpg" )
+Activity.create(name: "Base Jumping", description: "WHOOO.", image_url: "scuba.jpg" )
+Activity.create(name: "Hunting", description: "BANG.", image_url: "scuba.jpg" )
+
+Trip.create(destination_id: 1, activity_id: 1, total_cost: 20000.00, trip_cost: 10000.00, name: "Canadian Rockies" )
+Trip.create(destination_id: 6, activity_id: 3, total_cost: 21000.00, trip_cost: 21000.00, name: "Great Barrier Reef" )
+Trip.create(destination_id: 5, activity_id: 6, total_cost: 100000.00, trip_cost: 200000.00, name: "African Safari" )
+Trip.create(destination_id: 3, activity_id: 5, total_cost: 20000.00, trip_cost: 15000.00, name: "Swiss Base Jumpers" )
+Trip.create(destination_id: 4, activity_id: 4, total_cost: 10000.00, trip_cost: 8000.00, name: "Himalayan Mountain Climbing"  )
```

Comment:
Line is too long. [127/80]<br>Unnecessary spacing detected.<br>Space inside parentheses detected.